Describe license key errors (#5657)
DevExpress/devextreme-documentation: api-reference/50 Common/utils/zz Errors and Warnings/W0019.md

---
---
##### shortDescription

> Occurs when the valid license key is not found.

Reviewer: ```suggestion
Occurs when a valid license key cannot be found.
```
